.github/workflows/ubuntu-cminpack-install.yaml

```diff
@@ -12,20 +12,15 @@ jobs:
         BUILD_SHARED_LIBS: [ 'ON' , 'OFF' ]
         CMAKE_BUILD_TYPE: [ 'Release', 'Debug', 'RelWithDebInfo', 'MinSizeRel' ]
         CMINPACK_PRECISION: [ 'd' ] # seems to be the only precision that the current tests reference work
-        USE_BLAS: [ 'OFF' ]
```

Comment:
Honestly, I don't think we need to add a dimension to the matrix. Testing one configuration with BLAS is enough to me. This is why I did it that way for Ubuntu, and I would vote to do the same for Windows.

Reply:
It could be done / reverted to test a single combination of matrix configs. In my view, the chosen configuration could be the most common among users, or package maintainers (e.g.: shared library in Release mode).

My point in testing all configurations is that they are almost equivalent in terms of execution time (they finish almost all in the same time). So, I thought: why not? I understand that it could be a problem or unnecessary in the long run. It is just a matter of choice, really.

Reply:
This is the configuration that is currently tested is the following. I'm ok with changing RelWithDebInfo to Release
```
          - BUILD_SHARED_LIBS: 'ON'
            CMAKE_BUILD_TYPE: 'RelWithDebInfo'
            CMINPACK_PRECISION: 'd'
            USE_BLAS: 'ON'
```
Why not test all matrix configurations? It may be free, but it has some impact. Because building and testing requires energy, which may be non-renewable, and I still care about this planet.